extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -661,12 +661,14 @@ export class AmpA4A extends AMP.BaseElement {
                 this.protectedEmitLifecycleEvent_(eventName, extraVariables);
               })
               .then(status => {
-                this.protectedEmitLifecycleEvent_('adResponseValidateEnd');
                 if (getMode().localDev &&
                     this.element.getAttribute('type') == 'fake') {
                   // do not verify signature for fake type ad
                   status = VerificationStatus.OK;
                 }
+                this.protectedEmitLifecycleEvent_('adResponseValidateEnd', {
+                  result: status,
```

Comment:
There's precedent in the code for leaving them unquoted, and manual tests don't show any difference. Nevertheless, I quoted the props just in case.